en/manual/platforms/linux/setup-and-requirements.md

```diff
@@ -1,8 +1,8 @@
 # Setup and requirements
 
-To develop for Linux using Stride, you need a Linux PC with a graphics card that supports at least OpenGL 4.2 or Vulkan 1.0. The preferred Linux distribution for Stride is Ubuntu 16.04 or later, as this was the setup we used to develop the Linux version of Stride.
+To develop for Linux using Stride, you need a Linux PC with a graphics card that supports at least OpenGL 4.2 or Vulkan 1.0. The preferred Linux distribution for Stride is Debian 12 or later, as this was the setup we used to develop the Linux version of Stride.
 
-The instructions below assume you have Ubuntu 16.04 installed. You might need to adapt them according to your Linux distribution.
+The instructions below assume you have Debian 12 installed.
```

Comment:
This can also be updated to include Fedora. I can confirm these instructions work on Fedora 39.

Reply:
I can also confirm, because I was using in the past.  Anyway, It should doesn't really matter, Stride should work on any distro. Maybe should mention this?

Reply:
It should be mentioned that other distros _should_ work, but maybe we should have 1-3 that have some form of regular testing/checks by the community that they are indeed working. These would then provide users with a definite working distro.